src/test/components/__snapshots__/ProfileCallTreeView.test.js.snap

```diff
@@ -873,7 +873,7 @@ exports[`calltree/ProfileCallTreeView computes a width for a call tree of a real
             style={
               Object {
                 "height": "304px",
-                "width": 3120,
+                "width": 3130,
```

Comment:
I was guessing here that the 10 pixels weren't critical, so I updated the snapshot rather than subtracting one from `maxDepth` in the generating code in order to keep it simple.

Reply:
I even believe this is correct :)